COPYRIGHT.txt

```diff
@@ -293,6 +293,11 @@ Comment: jpeg-compressor
 Copyright: 2012, Rich Geldreich
 License: public-domain or Apache-2.0
 
+Files: ./thirdparty/libbacktrace/
```

Comment:
Would you mind moving this down below before `./thirdparty/libpng/` to keep them alphabetical